config/rbac/role.yaml

```diff
@@ -6,6 +6,12 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
Did you add these manually? If so, you could consider adding annotations to the `types_*.go` file. See https://book.kubebuilder.io/cronjob-tutorial/controller-implementation.html

Reply:
No this is an auto generated Kustomize/kubebuilder file. I don't follow? There is no types_*.go file

I added the // +kubebuilder comment to top of my Reconcile function (see 3rd comment below), Then kubebuilder auto-generates the RBAC files:

`// +kubebuilder:rbac:groups=plumber.k8snetworking.pf9.io,resources=networkplugins,verbs=get;list;watch;create;update;patch;delete
// +kubebuilder:rbac:groups=plumber.k8snetworking.pf9.io,resources=networkplugins/status,verbs=get;update;patch
// +kubebuilder:rbac:groups=*,resources=*,verbs=*

func (r *NetworkPluginsReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {`